modules/consensus/synchronize.go

```diff
@@ -10,22 +14,43 @@ import (
 	"github.com/NebulousLabs/bolt"
 )
 
+const (
+	// minNumOutbound is the minimum number of outbound peers required before ibd
+	// is confident we are synced.
+	minNumOutbound = 4
+)
+
 var (
 	// MaxCatchUpBlocks is the maxiumum number of blocks that can be given to
 	// the consensus set in a single iteration during the initial blockchain
 	// download.
 	MaxCatchUpBlocks = func() types.BlockHeight {
 		switch build.Release {
-		case "testing":
-			return 3
+		case "dev":
+			return 10
```

Comment:
I would almost make this higher for dev, since the dev constants will have smaller blocks and be on local networks.